### PR TITLE
CSP headers

### PR DESCRIPTION
**Pull request by Oluane**
## Description

This PR adds the [`Content-Security-Policy` header](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP) in all responses coming from our server, adding a layer of security concerning the prevention of XSS or data injection attacks. This issue has been pointed up by Lighthouse audit reports. 

There are several ways to integrate it with Nextjs, the first and simpler one was to add a `header` property in the `next.config.js` file, with the header name and its content. I first went this way, but I had issues with two different directives, who needed dynamic values to work, since Nextjs & Styled Components inject some scripts inline. 

So, to avoid using `unsafe-inline` value to the `script-src` and `style-src` directives, I decided to use the `nonce-{your-nonce-here}` solution to make the browser able to recognize inline scripts injected by "ourselves". The idea is that the server creates a nonce for each render, we tell Next to attach this nonce to each scripts, and we set this nonce inside the server's responses in the CSP header, so the browser checks if the scripts' nonce it received is the same as the one that is set in the response's headers. 
This way worked well for Next part, but not so good for SC one. There's no out-of-the-box way to give this nonce value to SC's scripts (like for Next, on `Head` and `NextScript` elements), but a kind of workaround with `webpack` that I tried. It worked, but with a big drawback : the style glitched. It was first blocked by the browser and not applied, then a flash and the style arrives. After digging in issues and posts, I found that there was no straightforward solution but to use `unsafe-inline` for the `style-src` directive for now. 

EDIT: I also set to true the production of code source map by Next in `next.config.ts` in this PR, after seeing that was missing in Lighthouse reports.

## Type of change

- [ ] **Chore** (non-breaking change which refactors / improves the existing code base).
- [ ] **Bug fix** (non-breaking change which fixes an issue).
- [x] **New feature** (non-breaking change which adds functionality).
- [ ] **Breaking change** (fix or feature that would cause existing functionality to not work as expected).

## Checklist

- [x] My code follows the style [**contributing guidelines**][contributing_file] of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.

[contributing_file]: https://github.com/fewlinesco/connect-account/blob/master/README.adoc

### CODE REVIEW

**Comment by frieda-ewlines** [comment]
<details><summary>Click here to see the Lighthouse manifest</summary><br><p><strong>/:</strong></p><ul><li>Performance: 0.91</li><li>Accessibility: 1</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/:</strong></p><ul><li>Performance: 0.96</li><li>Accessibility: 0.92</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/profile/:</strong></p><ul><li>Performance: 0.94</li><li>Accessibility: 0.91</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/profile/user-profile/new/:</strong></p><ul><li>Performance: 0.86</li><li>Accessibility: 0.9</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/:</strong></p><ul><li>Performance: 0.94</li><li>Accessibility: 0.9</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/email/new/:</strong></p><ul><li>Performance: 0.92</li><li>Accessibility: 0.9</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/phone/new/:</strong></p><ul><li>Performance: 0.93</li><li>Accessibility: 0.91</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/security/:</strong></p><ul><li>Performance: 0.95</li><li>Accessibility: 0.9</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/security/sudo/:</strong></p><ul><li>Performance: 0.88</li><li>Accessibility: 0.9</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/locale/:</strong></p><ul><li>Performance: 0.89</li><li>Accessibility: 0.92</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr></details>

**Comment by frieda-ewlines** [comment]
<details><summary>Click here to see the Lighthouse manifest</summary><br><p><strong>/:</strong></p><ul><li>Performance: 0.9</li><li>Accessibility: 1</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/:</strong></p><ul><li>Performance: 0.9</li><li>Accessibility: 0.92</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/profile/:</strong></p><ul><li>Performance: 0.87</li><li>Accessibility: 0.91</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/profile/user-profile/new/:</strong></p><ul><li>Performance: 0.83</li><li>Accessibility: 0.9</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/:</strong></p><ul><li>Performance: 0.91</li><li>Accessibility: 0.9</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/email/new/:</strong></p><ul><li>Performance: 0.85</li><li>Accessibility: 0.9</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/phone/new/:</strong></p><ul><li>Performance: 0.81</li><li>Accessibility: 0.91</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/security/:</strong></p><ul><li>Performance: 0.92</li><li>Accessibility: 0.9</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/security/sudo/:</strong></p><ul><li>Performance: 0.84</li><li>Accessibility: 0.9</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/locale/:</strong></p><ul><li>Performance: 0.83</li><li>Accessibility: 0.92</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr></details>

**Comment by frieda-ewlines** [comment]
<details><summary>Click here to see the Lighthouse manifest</summary><br><p><strong>/:</strong></p><ul><li>Performance: 0.9</li><li>Accessibility: 1</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/:</strong></p><ul><li>Performance: 0.96</li><li>Accessibility: 0.92</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/profile/:</strong></p><ul><li>Performance: 0.93</li><li>Accessibility: 0.91</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/profile/user-profile/new/:</strong></p><ul><li>Performance: 0.87</li><li>Accessibility: 0.9</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/:</strong></p><ul><li>Performance: 0.82</li><li>Accessibility: 0.9</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/email/new/:</strong></p><ul><li>Performance: 0.91</li><li>Accessibility: 0.9</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/phone/new/:</strong></p><ul><li>Performance: 0.88</li><li>Accessibility: 0.91</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/security/:</strong></p><ul><li>Performance: 0.86</li><li>Accessibility: 0.9</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/security/sudo/:</strong></p><ul><li>Performance: 0.9</li><li>Accessibility: 0.9</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/locale/:</strong></p><ul><li>Performance: 0.86</li><li>Accessibility: 0.92</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr></details>

**Comment by frieda-ewlines** [comment]
<details><summary>Click here to see the Lighthouse manifest</summary><br><p><strong>/:</strong></p><ul><li>Performance: 0.9</li><li>Accessibility: 1</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/:</strong></p><ul><li>Performance: 0.96</li><li>Accessibility: 0.92</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/profile/:</strong></p><ul><li>Performance: 0.93</li><li>Accessibility: 0.91</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/profile/user-profile/new/:</strong></p><ul><li>Performance: 0.86</li><li>Accessibility: 0.9</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/:</strong></p><ul><li>Performance: 0.94</li><li>Accessibility: 0.9</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/email/new/:</strong></p><ul><li>Performance: 0.92</li><li>Accessibility: 0.9</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/phone/new/:</strong></p><ul><li>Performance: 0.9</li><li>Accessibility: 0.91</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/security/:</strong></p><ul><li>Performance: 0.93</li><li>Accessibility: 0.9</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/security/sudo/:</strong></p><ul><li>Performance: 0.89</li><li>Accessibility: 0.9</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/locale/:</strong></p><ul><li>Performance: 0.88</li><li>Accessibility: 0.92</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr></details>

**Comment by frieda-ewlines** [comment]
<details><summary>Click here to see the Lighthouse manifest</summary><br><p><strong>/:</strong></p><ul><li>Performance: 0.89</li><li>Accessibility: 1</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/:</strong></p><ul><li>Performance: 0.93</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/profile/:</strong></p><ul><li>Performance: 0.92</li><li>Accessibility: 0.91</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/profile/user-profile/new/:</strong></p><ul><li>Performance: 0.81</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/:</strong></p><ul><li>Performance: 0.95</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/email/new/:</strong></p><ul><li>Performance: 0.89</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/phone/new/:</strong></p><ul><li>Performance: 0.87</li><li>Accessibility: 0.91</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/security/:</strong></p><ul><li>Performance: 0.93</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/security/sudo/:</strong></p><ul><li>Performance: 0.89</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/locale/:</strong></p><ul><li>Performance: 0.91</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr></details>

**Comment by frieda-ewlines** [comment]
<details><summary>Click here to see the Lighthouse manifest</summary><br><p><strong>/:</strong></p><ul><li>Performance: 0.89</li><li>Accessibility: 1</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/:</strong></p><ul><li>Performance: 0.95</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/profile/:</strong></p><ul><li>Performance: 0.93</li><li>Accessibility: 0.91</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/profile/user-profile/new/:</strong></p><ul><li>Performance: 0.85</li><li>Accessibility: 0.91</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/:</strong></p><ul><li>Performance: 0.93</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/email/new/:</strong></p><ul><li>Performance: 0.87</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/phone/new/:</strong></p><ul><li>Performance: 0.84</li><li>Accessibility: 0.91</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/security/:</strong></p><ul><li>Performance: 0.89</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/security/sudo/:</strong></p><ul><li>Performance: 0.89</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/locale/:</strong></p><ul><li>Performance: 0.91</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr></details>

**Comment by frieda-ewlines** [comment]
<details><summary>Click here to see the Lighthouse manifest</summary><br><p><strong>/:</strong></p><ul><li>Performance: 0.88</li><li>Accessibility: 1</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/:</strong></p><ul><li>Performance: 0.95</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/profile/:</strong></p><ul><li>Performance: 0.89</li><li>Accessibility: 0.91</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/profile/user-profile/new/:</strong></p><ul><li>Performance: 0.87</li><li>Accessibility: 0.91</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/:</strong></p><ul><li>Performance: 0.91</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/email/new/:</strong></p><ul><li>Performance: 0.84</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/phone/new/:</strong></p><ul><li>Performance: 0.82</li><li>Accessibility: 0.91</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/security/:</strong></p><ul><li>Performance: 0.92</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/security/sudo/:</strong></p><ul><li>Performance: 0.88</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/locale/:</strong></p><ul><li>Performance: 0.83</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr></details>

**Comment by frieda-ewlines** [comment]
<details><summary>Click here to see the Lighthouse manifest</summary><br><p><strong>/:</strong></p><ul><li>Performance: 0.87</li><li>Accessibility: 1</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/:</strong></p><ul><li>Performance: 0.86</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/profile/:</strong></p><ul><li>Performance: 0.84</li><li>Accessibility: 0.91</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/profile/user-profile/new/:</strong></p><ul><li>Performance: 0.87</li><li>Accessibility: 0.91</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/:</strong></p><ul><li>Performance: 0.94</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/email/new/:</strong></p><ul><li>Performance: 0.84</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/phone/new/:</strong></p><ul><li>Performance: 0.86</li><li>Accessibility: 0.91</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/security/:</strong></p><ul><li>Performance: 0.88</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/security/sudo/:</strong></p><ul><li>Performance: 0.91</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/locale/:</strong></p><ul><li>Performance: 0.88</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr></details>

**Comment by frieda-ewlines** [comment]
<details><summary>Click here to see the Lighthouse manifest</summary><br><p><strong>/:</strong></p><ul><li>Performance: 0.92</li><li>Accessibility: 1</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/:</strong></p><ul><li>Performance: 0.96</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/profile/:</strong></p><ul><li>Performance: 0.92</li><li>Accessibility: 0.91</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/profile/user-profile/new/:</strong></p><ul><li>Performance: 0.86</li><li>Accessibility: 0.91</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/:</strong></p><ul><li>Performance: 0.89</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/email/new/:</strong></p><ul><li>Performance: 0.89</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/phone/new/:</strong></p><ul><li>Performance: 0.88</li><li>Accessibility: 0.91</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/security/:</strong></p><ul><li>Performance: 0.92</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/security/sudo/:</strong></p><ul><li>Performance: 0.84</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/locale/:</strong></p><ul><li>Performance: 0.9</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr></details>

**Comment by frieda-ewlines** [comment]
<details><summary>Click here to see the Lighthouse manifest</summary><br><p><strong>/:</strong></p><ul><li>Performance: 0.93</li><li>Accessibility: 1</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/:</strong></p><ul><li>Performance: 0.92</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/profile/:</strong></p><ul><li>Performance: 0.93</li><li>Accessibility: 0.91</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/profile/user-profile/new/:</strong></p><ul><li>Performance: 0.86</li><li>Accessibility: 0.91</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/:</strong></p><ul><li>Performance: 0.94</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/email/new/:</strong></p><ul><li>Performance: 0.89</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/phone/new/:</strong></p><ul><li>Performance: 0.88</li><li>Accessibility: 0.91</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/security/:</strong></p><ul><li>Performance: 0.89</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/security/sudo/:</strong></p><ul><li>Performance: 0.91</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/locale/:</strong></p><ul><li>Performance: 0.89</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr></details>

**Comment by frieda-ewlines** [comment]
<details><summary>Click here to see the Lighthouse manifest</summary><br><p><strong>/:</strong></p><ul><li>Performance: 0.92</li><li>Accessibility: 1</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/:</strong></p><ul><li>Performance: 0.96</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/profile/:</strong></p><ul><li>Performance: 0.91</li><li>Accessibility: 0.91</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/profile/user-profile/new/:</strong></p><ul><li>Performance: 0.88</li><li>Accessibility: 0.91</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/:</strong></p><ul><li>Performance: 0.93</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/email/new/:</strong></p><ul><li>Performance: 0.9</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/phone/new/:</strong></p><ul><li>Performance: 0.89</li><li>Accessibility: 0.91</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/security/:</strong></p><ul><li>Performance: 0.9</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/security/sudo/:</strong></p><ul><li>Performance: 0.91</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/locale/:</strong></p><ul><li>Performance: 0.9</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr></details>

**Review by Fuzznimp — COMMENTED** [review]
Good for me. Nice work 😄 👍

**Comment by frieda-ewlines** [comment]
<details><summary>Click here to see the Lighthouse manifest</summary><br><p><strong>/:</strong></p><ul><li>Performance: 0.87</li><li>Accessibility: 1</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/:</strong></p><ul><li>Performance: 0.91</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/profile/:</strong></p><ul><li>Performance: 0.93</li><li>Accessibility: 0.91</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/profile/user-profile/new/:</strong></p><ul><li>Performance: 0.87</li><li>Accessibility: 0.91</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/:</strong></p><ul><li>Performance: 0.95</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/email/new/:</strong></p><ul><li>Performance: 0.91</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/phone/new/:</strong></p><ul><li>Performance: 0.92</li><li>Accessibility: 0.91</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/security/:</strong></p><ul><li>Performance: 0.93</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/security/sudo/:</strong></p><ul><li>Performance: 0.9</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/locale/:</strong></p><ul><li>Performance: 0.92</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr></details>

**Comment by frieda-ewlines** [comment]
<details><summary>Click here to see the Lighthouse manifest</summary><br><p><strong>/:</strong></p><ul><li>Performance: 0.91</li><li>Accessibility: 1</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/:</strong></p><ul><li>Performance: 0.81</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/profile/:</strong></p><ul><li>Performance: 0.9</li><li>Accessibility: 0.91</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/profile/user-profile/new/:</strong></p><ul><li>Performance: 0.84</li><li>Accessibility: 0.91</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/:</strong></p><ul><li>Performance: 0.93</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/email/new/:</strong></p><ul><li>Performance: 0.89</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/phone/new/:</strong></p><ul><li>Performance: 0.9</li><li>Accessibility: 0.91</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/security/:</strong></p><ul><li>Performance: 0.92</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/security/sudo/:</strong></p><ul><li>Performance: 0.95</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/locale/:</strong></p><ul><li>Performance: 0.85</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr></details>

File: next.config.js [hunk 1]
```diff
@@ -33,6 +33,7 @@ const moduleExports = {
 
     return config;
   },
+  productionBrowserSourceMaps: true,
```
Comment (Fenntasy) [inline next.config.js:36]: you forgot that in your PR summary 😁

**Review by Fenntasy — APPROVED** [review]

File: next.config.js (hunk 1 above)
Comment (Oluane) [reply next.config.js:36]: oops 😬

**Review by Fenntasy — APPROVED** [review]

**Comment by frieda-ewlines** [comment]
<details><summary>Click here to see the Lighthouse manifest</summary><br><p><strong>/:</strong></p><ul><li>Performance: 0.93</li><li>Accessibility: 1</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/:</strong></p><ul><li>Performance: 0.95</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/profile/:</strong></p><ul><li>Performance: 0.87</li><li>Accessibility: 0.93</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/profile/user-profile/new/:</strong></p><ul><li>Performance: 0.85</li><li>Accessibility: 0.93</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/:</strong></p><ul><li>Performance: 0.9</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/email/new/:</strong></p><ul><li>Performance: 0.91</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/phone/new/:</strong></p><ul><li>Performance: 0.88</li><li>Accessibility: 0.92</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/security/:</strong></p><ul><li>Performance: 0.9</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/security/sudo/:</strong></p><ul><li>Performance: 0.91</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/locale/:</strong></p><ul><li>Performance: 0.89</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr></details>

**Comment by frieda-ewlines** [comment]
Task linked: [CU-1cdcwde [improve] - Adding Content Security Policy in responses headers](https://app.clickup.com/t/1cdcwde)

**Pull request merged**